addon/components/challenges/issue.js

```diff
@@ -3,7 +3,25 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   layout,
-  isPending: false,
-  isFailure: false,
-  isSucces: false
+  getTitle: Ember.computed('model', function() {
```

Comment:
There is actually a css method for this: https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/